platform/app/public/html-templates/index.html

```diff
@@ -214,6 +214,12 @@
       rel="preload"
       as="style"
     />
+    <script>
```

Comment:
This is the work around for the dynamic import - just use the browser native import.